v2/yarpcprotobuf/protoc-gen-yarpc-go/internal/generator/registry.go

```diff
@@ -99,14 +107,17 @@ func (r *registry) Load(req *plugin.CodeGeneratorRequest) error {
 				return err
 			}
 		}
+		if err := r.loadDependencies(target); err != nil {
+			return err
+		}
 	}
 	return nil
 }
 
 // GetData returns the template data the corresponds
 // to the given filename.
 func (r *registry) GetData(filename string) (*Data, error) {
```

Comment:
Yea, I originally had this as exported so that it would describe that this is the entry point to the private `registry` struct's functionality. In other words, this is the only method called from a function/method in another type (even though it's invoked from within the same package).

Reply:
sgtm